web/templates/skipped_marking_unread.hbs

```diff
@@ -0,0 +1,23 @@
+{{#if (eq stream_names_with_privacy_symbol_html.length 1)}}
+    {{#tr}}
+    Because you are not subscribed to <z-stream></z-stream>, messages in this channel were not marked as unread.
+    {{#*inline "z-stream"}}<strong class="white-space-nowrap">{{{stream_names_with_privacy_symbol_html.[0]}}}</strong>{{/inline}}
+    {{/tr}}
+{{else}}
+    {{#tr}}
+        Because you are not subscribed to <z-streams></z-streams>, messages in these channels were not marked as unread.
+        {{#*inline "z-streams"}}
+            {{~#each stream_names_with_privacy_symbol_html as |stream index|~}}
+                {{~#unless @first~}}
+                    {{~#unless @last~}}
+                        <span>,</span>
+                    {{/unless~}}
+                    {{~#if @last~}}
+                        <span> and </span>
+                    {{/if~}}
+                {{/unless~}}
```

Comment:
This isn't properly internationalized; it should be possible to use `format_array_as_list` in the caller to format the list of channels properly.